utils/src/ooinstall/cli_installer.py

```diff
@@ -562,6 +569,10 @@ def get_missing_info_from_user(oo_cfg):
         oo_cfg.settings['master_routingconfig_subdomain'] = get_master_routingconfig_subdomain()
         click.clear()
 
+    if not oo_cfg.settings.get('master_routingconfig_subdomain', None):
+        oo_cfg.settings['master_routingconfig_subdomain'] = get_proxy_hostname()
```

Comment:
Isn't the `master_routingconfig_subdomain` something totally different from the proxy?  The latest docs PR shows the inventory values that should be set https://github.com/openshift/openshift-docs/pull/1607/files

For a-o-i I think we'd only want to set a single proxy hostname for all the proxy values and then allow them to set excludes.